org.eclipse.jdt.core.manipulation/core extension/org/eclipse/jdt/internal/corext/fix/Java50CleanUpCore.java

```diff
@@ -0,0 +1,203 @@
+/*******************************************************************************
```

Comment:
Looking at other similar changes, we didn't migrate the actual clean up to core manipulation. The reason is that most of the functionality this adds onto `Java50Fix` is stuff that's only helpful in the context of configuring the clean up using a UI. I'll put this class back into `eclipse.jdt.ui` when I get a chance.